CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+### 0.41.26
+- Support for passing additional items to the navbar for `Tabs`
+- Updated React peer dependency version
```

Comment:
Consistent tense and grammar here too, please.
“Add support for...”, and “Update React...” and so on.

Reply:
done